src/minesweeper/use-cases/startGame/startGame.validations.ts

```diff
@@ -1,23 +1,30 @@
 import { MinesweeperError } from '../validations';
 
-export enum ErrorStartGame {
+export enum StartGameErrorMessages {
   BOMBS_GREATER_THAN_ZERO = 'bombs must be greater than 0',
   ROWS_GREATER_THAN_ZERO = 'rows must be greater than 0',
   COLUMNS_GREATER_THAN_ZERO = 'columns must be greater than 0',
-  ROWS_AND_COLUMNS_GREATER_THAN_ONE = 'rows and columns must be greater than 1',
-  BOMBS_GREATER_THAN_TOTAL_CELLS = 'bombs must be less than total cells',
+  BOMBS_LESS_THAN_TOTAL_CELLS = 'bombs must be less than total cells',
+  BOMBS_IS_INTEGER = 'bombs must be a integer number',
+  ROWS_IS_INTEGER = 'rows must be a integer number',
```

Comment:
Is not integer, is a natural number. `Integer` includes negative numbers

Reply:
yep, but the other validation only accepts values greater than 1